database/migrations/2019_05_11_144019_add_category_rooms.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddCategoryRooms extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('rooms', function (Blueprint $table) {
+            $table->integer('category_id')->unsigned();
```

Comment:
@tpharaoh here I would add `->nullable()` and also add a `$table->foreign()` to form proper DB-level foreign key and not just the column. So fix this and I will happily merge your Pull Request. Thanks!

Reply:
done in latest commit